src/main/drivers/accgyro/accgyro_mpu.c

```diff
@@ -58,6 +58,8 @@
 #include "drivers/accgyro/accgyro_spi_l3gd20.h"
 #include "drivers/accgyro/accgyro_mpu.h"
 
+#include "sensors/gyro_init.h"
```

Comment:
The driver layer should not have up references to the firmware layer - also, this isn't needed. :wink:

Reply:
An easy fix then! Done.